pyoozie/model.py

```diff
@@ -0,0 +1,587 @@
+# Copyright (c) 2017 "Shopify inc." All rights reserved.
```

Comment:
Could we move this module under `pyoozie/api/_model.py` to make it private and put it under `api` to reduce ambiguity?